src/main/java/com/google/sps/servlets/utils/Information.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets.utils;
+
+import com.google.api.services.youtube.model.Video;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.VideoSnippet;
+import com.google.api.services.youtube.model.VideoStatistics;
+
+public class Information {
```

Comment:
Nit: Maybe name this something more specific such as VideoInformation?

Reply:
+1